### DISCUSSION TITLE
Future of MediatR and .Net Framework

### DISCUSSION THREAD

**Discussion by TheNephalim** [LuckyPennySoftware/MediatR#720] [General]
Because of *insert reason here*, I'm still running .NET Framework 4.8 on my projects.  I needed to update MediatR in an older project and found it would not install because of the message:  

Package MediatR 10.0.1 is not compatible with net48 (.NETFramework,Version=v4.8) / win. Package MediatR 10.0.1 supports: netstandard2.1 (.NETStandard,Version=v2.1)

After looking on NuGet, it appears that since version 10.0.0, MediatR is no longer targeting the .Net Framework.  

I just wanted to confirm this is the case.

**Comment by jbogard** [comment]
Yes new versions do not target the .NET Framework.

**Reply by TheNephalim** [reply]
I guess I will need to keep it at version 9.0.0 until I get the time to convert the application(s) to .NET 6.

Thanks for the confirmation and the library.
